src/main/java/uk/gov/hmcts/cmc/claimstore/services/ccd/callbacks/Callback.java

```diff
@@ -0,0 +1,7 @@
+package uk.gov.hmcts.cmc.claimstore.services.ccd.callbacks;
+
+import uk.gov.hmcts.reform.ccd.client.model.CallbackResponse;
+
+public interface Callback {
```

Comment:
Any value in this being called out as `@FunctionalInterface`?

Reply:
mmm... not really with this design :) will give it a thought